From 8474b70c3a3842cdf3d51f331dd117ab6421f6d0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Thu, 25 Sep 2014 17:27:46 -0400
Subject: [PATCH] man: use <constant> markup for errno value

We were inconsistent, but marking them up as constants makes more
sense then as variables.
---
 man/sd_bus_creds_get_pid.xml              | 10 +++++-----
 man/sd_bus_creds_new_from_pid.xml         |  6 +++---
 man/sd_bus_error.xml                      |  4 ++--
 man/sd_bus_message_append_basic.xml       | 10 +++++-----
 man/sd_bus_message_get_cookie.xml         |  4 ++--
 man/sd_bus_message_get_monotonic_usec.xml |  4 ++--
 man/sd_bus_negotiate_fds.xml              |  2 +-
 man/sd_bus_new.xml                        |  2 +-
 man/sd_bus_open_user.xml                  |  4 ++--
 man/sd_bus_request_name.xml               | 14 +++++++-------
 man/sd_event_add_child.xml                | 10 +++++-----
 man/sd_event_add_defer.xml                |  8 ++++----
 man/sd_event_add_signal.xml               | 10 +++++-----
 man/sd_event_add_time.xml                 | 10 +++++-----
 man/sd_event_new.xml                      |  4 ++--
 15 files changed, 51 insertions(+), 51 deletions(-)

diff --git a/man/sd_bus_creds_get_pid.xml b/man/sd_bus_creds_get_pid.xml
index 40ed81ecb0..6358762470 100644
--- a/man/sd_bus_creds_get_pid.xml
+++ b/man/sd_bus_creds_get_pid.xml
@@ -381,7 +381,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
 
     <variablelist>
       <varlistentry>
-        <term><varname>-ENODATA</varname></term>
+        <term><constant>-ENODATA</constant></term>
 
         <listitem><para>Given field is not available in
         <parameter>c</parameter>.</para>
@@ -389,7 +389,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ENOENT</varname></term>
+        <term><constant>-ENOENT</constant></term>
 
         <listitem><para>Given field is not specified for the sender.
         This will be returned by <function>sd_bus_get_unit</function>,
@@ -403,7 +403,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ENXIO</varname></term>
+        <term><constant>-ENXIO</constant></term>
 
         <listitem><para>An error occurred in parsing cgroup paths.
         <filename>libsystemd</filename> might be out of sync with
@@ -411,14 +411,14 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-EINVAL</varname></term>
+        <term><constant>-EINVAL</constant></term>
 
         <listitem><para>Specified pointer parameter is <constant>NULL</constant>.
         </para></listitem>
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ENOMEM</varname></term>
+        <term><constant>-ENOMEM</constant></term>
 
         <listitem><para>Memory allocation failed.</para></listitem>
       </varlistentry>
diff --git a/man/sd_bus_creds_new_from_pid.xml b/man/sd_bus_creds_new_from_pid.xml
index bc94c44095..f3ea1defe1 100644
--- a/man/sd_bus_creds_new_from_pid.xml
+++ b/man/sd_bus_creds_new_from_pid.xml
@@ -205,14 +205,14 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
     <variablelist>
 
       <varlistentry>
-        <term><varname>-ESRCH</varname></term>
+        <term><constant>-ESRCH</constant></term>
 
         <listitem><para>Specified <parameter>pid</parameter> could not
         be found.</para></listitem>
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-EINVAL</varname></term>
+        <term><constant>-EINVAL</constant></term>
 
         <listitem><para>Specified parameter is invalid
         (<constant>NULL</constant> in case of output
@@ -220,7 +220,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ENOMEM</varname></term>
+        <term><constant>-ENOMEM</constant></term>
 
         <listitem><para>Memory allocation failed.</para></listitem>
       </varlistentry>
diff --git a/man/sd_bus_error.xml b/man/sd_bus_error.xml
index 7144f4ba32..aba80832d2 100644
--- a/man/sd_bus_error.xml
+++ b/man/sd_bus_error.xml
@@ -376,7 +376,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
     <variablelist>
 
       <varlistentry>
-        <term><varname>-EINVAL</varname></term>
+        <term><constant>-EINVAL</constant></term>
 
         <listitem><para>Error was already set in
         <structname>sd_bus_error</structname> structure when one the
@@ -384,7 +384,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ENOMEM</varname></term>
+        <term><constant>-ENOMEM</constant></term>
 
         <listitem><para>Memory allocation failed.</para></listitem>
       </varlistentry>
diff --git a/man/sd_bus_message_append_basic.xml b/man/sd_bus_message_append_basic.xml
index 7c5f1e9ddc..2594134d87 100644
--- a/man/sd_bus_message_append_basic.xml
+++ b/man/sd_bus_message_append_basic.xml
@@ -219,35 +219,35 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
     <variablelist>
 
       <varlistentry>
-        <term><varname>-EINVAL</varname></term>
+        <term><constant>-EINVAL</constant></term>
 
         <listitem><para>Specified parameter is invalid.
         </para></listitem>
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-EPERM</varname></term>
+        <term><constant>-EPERM</constant></term>
 
         <listitem><para>Message has been sealed.
         </para></listitem>
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ESTALE</varname></term>
+        <term><constant>-ESTALE</constant></term>
 
         <listitem><para>Message is in invalid state.
         </para></listitem>
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ENXIO</varname></term>
+        <term><constant>-ENXIO</constant></term>
 
         <listitem><para>Message cannot be appended to.
         </para></listitem>
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ENOMEM</varname></term>
+        <term><constant>-ENOMEM</constant></term>
 
         <listitem><para>Memory allocation failed.</para></listitem>
       </varlistentry>
diff --git a/man/sd_bus_message_get_cookie.xml b/man/sd_bus_message_get_cookie.xml
index 641ecec659..3e3f9bd7be 100644
--- a/man/sd_bus_message_get_cookie.xml
+++ b/man/sd_bus_message_get_cookie.xml
@@ -107,14 +107,14 @@
 
                 <variablelist>
                         <varlistentry>
-                                <term><varname>-EINVAL</varname></term>
+                                <term><constant>-EINVAL</constant></term>
 
                                 <listitem><para>A specified parameter
                                 is invalid.</para></listitem>
                         </varlistentry>
 
                         <varlistentry>
-                                <term><varname>-ENODATA</varname></term>
+                                <term><constant>-ENODATA</constant></term>
 
                                 <listitem><para>No cookie has been
                                 assigned to this message. This either
diff --git a/man/sd_bus_message_get_monotonic_usec.xml b/man/sd_bus_message_get_monotonic_usec.xml
index 311ba55619..290faf2a5b 100644
--- a/man/sd_bus_message_get_monotonic_usec.xml
+++ b/man/sd_bus_message_get_monotonic_usec.xml
@@ -139,14 +139,14 @@
 
                 <variablelist>
                         <varlistentry>
-                                <term><varname>-EINVAL</varname></term>
+                                <term><constant>-EINVAL</constant></term>
 
                                 <listitem><para>A specified parameter
                                 is invalid.</para></listitem>
                         </varlistentry>
 
                         <varlistentry>
-                                <term><varname>-ENODATA</varname></term>
+                                <term><constant>-ENODATA</constant></term>
 
                                 <listitem><para>No timestamp or
                                 sequence number information is
diff --git a/man/sd_bus_negotiate_fds.xml b/man/sd_bus_negotiate_fds.xml
index c91318cd9d..fb313e34fb 100644
--- a/man/sd_bus_negotiate_fds.xml
+++ b/man/sd_bus_negotiate_fds.xml
@@ -143,7 +143,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
 
     <variablelist>
       <varlistentry>
-        <term><varname>-EPERM</varname></term>
+        <term><constant>-EPERM</constant></term>
 
         <listitem><para>The bus connection has already been started.</para></listitem>
       </varlistentry>
diff --git a/man/sd_bus_new.xml b/man/sd_bus_new.xml
index 1f8d787eea..8c56dc0d20 100644
--- a/man/sd_bus_new.xml
+++ b/man/sd_bus_new.xml
@@ -112,7 +112,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
 
     <variablelist>
       <varlistentry>
-        <term><varname>-ENOMEM</varname></term>
+        <term><constant>-ENOMEM</constant></term>
 
         <listitem><para>Memory allocation failed.</para></listitem>
       </varlistentry>
diff --git a/man/sd_bus_open_user.xml b/man/sd_bus_open_user.xml
index 6cb4a314a5..47f8361486 100644
--- a/man/sd_bus_open_user.xml
+++ b/man/sd_bus_open_user.xml
@@ -171,7 +171,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
     <variablelist>
 
       <varlistentry>
-        <term><varname>-EINVAL</varname></term>
+        <term><constant>-EINVAL</constant></term>
 
         <listitem><para>Specified parameter is invalid
         (<constant>NULL</constant> in case of output
@@ -179,7 +179,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ENOMEM</varname></term>
+        <term><constant>-ENOMEM</constant></term>
 
         <listitem><para>Memory allocation failed.</para></listitem>
       </varlistentry>
diff --git a/man/sd_bus_request_name.xml b/man/sd_bus_request_name.xml
index c23bbb37bc..ca082183cc 100644
--- a/man/sd_bus_request_name.xml
+++ b/man/sd_bus_request_name.xml
@@ -143,7 +143,7 @@
 
                 <variablelist>
                         <varlistentry>
-                                <term><varname>-EALREADY</varname></term>
+                                <term><constant>-EALREADY</constant></term>
 
                                 <listitem><para>The caller already is
                                 the owner of the specified
@@ -151,7 +151,7 @@
                         </varlistentry>
 
                         <varlistentry>
-                                <term><varname>-EEXIST</varname></term>
+                                <term><constant>-EEXIST</constant></term>
 
                                 <listitem><para>The name has already
                                 been acquired by a different peer, and
@@ -163,7 +163,7 @@
                         </varlistentry>
 
                         <varlistentry>
-                                <term><varname>-ESRCH</varname></term>
+                                <term><constant>-ESRCH</constant></term>
 
                                 <listitem><para>It was attempted to
                                 release a name that is currently not
@@ -172,7 +172,7 @@
                         </varlistentry>
 
                         <varlistentry>
-                                <term><varname>-EADDRINUSE</varname></term>
+                                <term><constant>-EADDRINUSE</constant></term>
 
                                 <listitem><para>It was attempted to
                                 release a name that is owned by a
@@ -181,21 +181,21 @@
                         </varlistentry>
 
                         <varlistentry>
-                                <term><varname>-EINVAL</varname></term>
+                                <term><constant>-EINVAL</constant></term>
 
                                 <listitem><para>A specified parameter
                                 is invalid.</para></listitem>
                         </varlistentry>
 
                         <varlistentry>
-                                <term><varname>-ENOTCONN</varname></term>
+                                <term><constant>-ENOTCONN</constant></term>
 
                                 <listitem><para>The bus connection has
                                 been disconnected.</para></listitem>
                         </varlistentry>
 
                         <varlistentry>
-                                <term><varname>-ECHILD</varname></term>
+                                <term><constant>-ECHILD</constant></term>
 
                                 <listitem><para>The bus connection has
                                 been created in a different process
diff --git a/man/sd_event_add_child.xml b/man/sd_event_add_child.xml
index f282a5094a..012fdb6a1a 100644
--- a/man/sd_event_add_child.xml
+++ b/man/sd_event_add_child.xml
@@ -137,13 +137,13 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
 
     <variablelist>
       <varlistentry>
-        <term><varname>-ENOMEM</varname></term>
+        <term><constant>-ENOMEM</constant></term>
 
         <listitem><para>Not enough memory to allocate an object.</para></listitem>
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-EINVAL</varname></term>
+        <term><constant>-EINVAL</constant></term>
 
         <listitem><para>An invalid argument has been passed. This includes
         specyfing an empty mask in <parameter>options</parameter> or a mask
@@ -155,7 +155,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-EBUSY</varname></term>
+        <term><constant>-EBUSY</constant></term>
 
         <listitem><para>An handler is already installed for this
         child.</para></listitem>
@@ -163,14 +163,14 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ESTALE</varname></term>
+        <term><constant>-ESTALE</constant></term>
 
         <listitem><para>The event loop is already terminated.</para></listitem>
 
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ECHILD</varname></term>
+        <term><constant>-ECHILD</constant></term>
 
         <listitem><para>The event loop has been created in a different process.</para></listitem>
 
diff --git a/man/sd_event_add_defer.xml b/man/sd_event_add_defer.xml
index 6c937098b2..442600a93e 100644
--- a/man/sd_event_add_defer.xml
+++ b/man/sd_event_add_defer.xml
@@ -138,25 +138,25 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
 
     <variablelist>
       <varlistentry>
-        <term><varname>-ENOMEM</varname></term>
+        <term><constant>-ENOMEM</constant></term>
 
         <listitem><para>Not enough memory to allocate an object.</para></listitem>
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-EINVAL</varname></term>
+        <term><constant>-EINVAL</constant></term>
 
         <listitem><para>An invalid argument has been passed.</para></listitem>
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ESTALE</varname></term>
+        <term><constant>-ESTALE</constant></term>
 
         <listitem><para>The event loop is already terminated.</para></listitem>
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ECHILD</varname></term>
+        <term><constant>-ECHILD</constant></term>
 
         <listitem><para>The event loop has been created in a different process.</para></listitem>
       </varlistentry>
diff --git a/man/sd_event_add_signal.xml b/man/sd_event_add_signal.xml
index 1517c1021c..6584268818 100644
--- a/man/sd_event_add_signal.xml
+++ b/man/sd_event_add_signal.xml
@@ -134,20 +134,20 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
 
     <variablelist>
       <varlistentry>
-        <term><varname>-ENOMEM</varname></term>
+        <term><constant>-ENOMEM</constant></term>
 
         <listitem><para>Not enough memory to allocate an object.</para></listitem>
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-EINVAL</varname></term>
+        <term><constant>-EINVAL</constant></term>
 
         <listitem><para>An invalid argument has been passed.</para></listitem>
 
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-EBUSY</varname></term>
+        <term><constant>-EBUSY</constant></term>
 
         <listitem><para>An handler is already installed for this
         signal or the signal was not blocked previously.</para></listitem>
@@ -155,14 +155,14 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ESTALE</varname></term>
+        <term><constant>-ESTALE</constant></term>
 
         <listitem><para>The event loop is already terminated.</para></listitem>
 
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ECHILD</varname></term>
+        <term><constant>-ECHILD</constant></term>
 
         <listitem><para>The event loop has been created in a different process.</para></listitem>
 
diff --git a/man/sd_event_add_time.xml b/man/sd_event_add_time.xml
index f16b84f605..77b8828bb3 100644
--- a/man/sd_event_add_time.xml
+++ b/man/sd_event_add_time.xml
@@ -196,34 +196,34 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
 
     <variablelist>
       <varlistentry>
-        <term><varname>-ENOMEM</varname></term>
+        <term><constant>-ENOMEM</constant></term>
 
         <listitem><para>Not enough memory to allocate an object.</para></listitem>
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-EINVAL</varname></term>
+        <term><constant>-EINVAL</constant></term>
 
         <listitem><para>An invalid argument has been passed.</para></listitem>
 
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ESTALE</varname></term>
+        <term><constant>-ESTALE</constant></term>
 
         <listitem><para>The event loop is already terminated.</para></listitem>
 
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ECHILD</varname></term>
+        <term><constant>-ECHILD</constant></term>
 
         <listitem><para>The event loop has been created in a different process.</para></listitem>
 
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-ENOTSUP</varname></term>
+        <term><constant>-ENOTSUP</constant></term>
 
         <listitem><para>The selected clock is not supported by the event loop implementation.</para></listitem>
 
diff --git a/man/sd_event_new.xml b/man/sd_event_new.xml
index 3062f43e69..d392ed30b0 100644
--- a/man/sd_event_new.xml
+++ b/man/sd_event_new.xml
@@ -138,13 +138,13 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
 
     <variablelist>
       <varlistentry>
-        <term><varname>-ENOMEM</varname></term>
+        <term><constant>-ENOMEM</constant></term>
 
         <listitem><para>Not enough memory to allocate the object.</para></listitem>
       </varlistentry>
 
       <varlistentry>
-        <term><varname>-EMFILE</varname></term>
+        <term><constant>-EMFILE</constant></term>
 
         <listitem><para>The maximum number of event loops has been allocated.</para></listitem>
 
